public/src/js/widgets/columns/fronts-config.html

```diff
@@ -264,6 +264,16 @@
             <input type="checkbox" data-bind="
                 checked: meta.uneditable" />
 
+            <div class="platform-edit">
+                <span>Platform: </span><span data-bind="text: getPlatform()"></span>
+                <!-- ko if: !thisIsPlatformSpecificCollection() && parents().length <= 1 -->
+                <span><button class="tool" data-bind="click: $parents[1].splitCollection.bind($parents[1], meta, parents)">Split into app/web</button></span>
+                <!-- /ko -->
+                <!-- ko if: thisIsPlatformSpecificCollection() && $parents[1].isUniqueCollection(meta.displayName()) -->
+                <span><button class="tool" data-bind="click: $parents[1].changeToAnyPlatform.bind($parents[1], meta)">Change to any</button></span>
```

Comment:
Should we display and disable this button if the collection is not a unique collection for clarity?